docs/guides/latest/backup-multiple-target/overview.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
- Rename the folder to `batch-backup`
- There will be two doc in this folder.
	- 1. Overview (explain how it does work)
	- 2. Batch Backup (Backup different types of target with single backup-batch)